app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -46,6 +46,13 @@
    </select>
    <input type="text" name="location" placeholder="VC URL/Location">
 </div>
+
+<div class="form_row">
+    <label for="event_img">Image: </label>
+    <input type="file" id="event_img" name="event_img"
+          accept="image/png, image/jpeg">
```

Comment:
Where is the image uploaded to?

Reply:
It's not really uploaded yet. It is sent as formData (a dict with pairs of data and info) and I added enctype="multipart/form-data" to the form to enable the form to process the file data.
I am saving the picture data later when I process the image, resize.it, and remove the metadata.